ci-operator/step-registry/hypershift/none/ibmcloud/zvsi/OWNERS

```diff
@@ -0,0 +1,9 @@
+approvers:
```

Comment:
please replace this list with ibmcloud/zvsi owners

Reply:
Same for all owners files

Reply:
Initially I have kept our team members RH ids but the bot posted a comment that the members present in the OWNERS files are not part of this project so removed them. So please suggest what to in this case ?